kiwi/models/model.py

```diff
@@ -206,14 +207,15 @@ def get_mask(self, batch, output):
 
     @staticmethod
     def create_from_file(path):
-        model_dict = torch.load(
-            str(path), map_location=lambda storage, loc: storage
-        )
+
+        model_dict = load_torch_file(path)
         for model_name in Model.subclasses:
             if model_name in model_dict:
                 model = Model.subclasses[model_name].from_dict(model_dict)
                 return model
-        return None
+
+        # If no model is found
+        raise KeyError('No valid model data found in {}'.format(path))
```

Comment:
Nice, I like that you give a clear error message - I would have saved some time if we had done that earlier :)